src/pages/applicants/project-grants/thank-you.tsx

```diff
@@ -0,0 +1,56 @@
+import { Center, Heading, Link, Stack } from '@chakra-ui/react';
+import type { NextPage } from 'next';
+import Head from 'next/head';
+
+import { PageSubheading, PageText, PlaceholderImage } from '../../../components/UI';
+import { ESP_EMAIL_ADDRESS } from '../../../constants';
+
+const ProjectGrants: NextPage = () => {
+  return (
+    <>
+      <Head>
+        <title>Ethereum Ecosystem Program | Thank you for applying to Project Grants</title>
+        <meta name='description' content='Project Grants' />
```

Comment:
Should we make this page as `noindex` for SEO purposes? https://github.com/ethereum/esp-website/blob/b9b6721f6ded83d976311ba08723c4cbb0e29680/src/pages/grantee-finance-form.js#L158

Reply:
We should make it non-indexable, yep